src/qt/pivx/sendconfirmdialog.h

```diff
@@ -37,10 +36,10 @@ class TxDetailDialog : public QDialog
     void setDisplayUnit(int unit){this->nDisplayUnit = unit;};
 
 public Q_SLOTS:
-    void acceptTx();
+    void accept() override;
+    void close();
```

Comment:
I thought so too, but the compiler complained.
That's probably because the `QWidget::close()` function (inherited by `QDialog`) returns a boolean value, not void.
Actually, have a point of discussion about this: I think it would be better to not use `close()` at all here, but rather resort to `QDialog::reject()` and override that one.

Reply:
yeah but we should double check something. The `reset` method is the one that sets the return value to rejected but it hides the dialog, it doesn't close it.
QT documentation is not clear over the difference between hide and close.

Reply:
both `accept` and `reject` call `QDialog::done()` and Qt docs state:
> If this dialog is shown with exec(), done() also causes the local event loop to finish, and exec() to return r.
>
> As with QWidget::close(), done() deletes the dialog if the Qt::WA_DeleteOnClose flag is set. If the dialog is the application's main widget, the application terminates. If the dialog is the last window closed, the QApplication::lastWindowClosed() signal is emitted.

Reply:
k then, all good 🕺

Reply:
Added 26b5dc2e855232713a07fe94fbe010f7a4e6cc74 with the change